src/main/java/com/powsybl/openloadflow/sa/WoodburyDcSecurityAnalysis.java

```diff
@@ -79,20 +79,40 @@ protected DcLoadFlowParameters createParameters(LoadFlowParameters lfParameters,
      */
     private double[] calculatePostContingencyStatesForAContingency(DcLoadFlowContext loadFlowContext, DenseMatrix contingenciesStates, double[] flowStates,
                                                                    PropagatedContingency contingency, Map<String, ComputedContingencyElement> contingencyElementByBranch,
-                                                                   Set<LfBus> disabledBuses, Set<String> elementsToReconnect, ReportNode reportNode, Set<LfBranch> partialDisabledBranches) {
+                                                                   Set<LfBus> disabledBuses, Set<String> elementsToReconnect, Set<LfBranch> partialDisabledBranches, ReportNode reportNode) {
+        return calculatePostContingencyStatesForAContingency(loadFlowContext, contingenciesStates, flowStates, contingency, contingencyElementByBranch, disabledBuses, elementsToReconnect,
+                partialDisabledBranches, List.of(), new HashMap<>(), new DenseMatrix(0, 0), reportNode);
+    }
```

Comment:
Can you define somewhere en EmptyDenseMatrice that would created only once ?
As seen in https://github.com/powsybl/powsybl-open-loadflow/pull/1109
DenseMatrix should not be created frequently and should be reused as much as possible.
Because the native memory they allocate as a direct buffer is not efficiently manager by the Java Garbage Collector.
Even if it is a small one here, it would be better to use a singleton.

Reply:
Yes you are right. Thank you.
I created the singleton in `EmptyDenseMatrix` static class (in `WoodburyEngine`). 
To be clean, I guess this empty matrix should be moved in `DenseMatrix` class. What do you think @vidaldid-rte ?

Reply:
Work in progress [here](https://github.com/powsybl/powsybl-core/pull/3216).